packages/manager/apps/hub/src/index.scss

```diff
@@ -1 +1,80 @@
+@tailwind components;
 @tailwind utilities;
+
+h1,
+h2 {
+  margin-top: 1rem;
+}
+
+$ovh-sidebar-width: 18.75rem;
+$device-breakpoint-medium-max-width: 1279px;
+
+ovh-manager-banner-text .notification-banner {
+  padding-right: $ovh-sidebar-width !important;
+}
+
+@media screen and (min-width: $device-breakpoint-medium-max-width) {
+  .hub-main-view_sidebar_expanded {
+    margin-right: $ovh-sidebar-width;
+    max-width: calc(100% - $ovh-sidebar-width);
```

Comment:
**question:** Do we have the desired view for Mobiles? If yes, does this accommodate that?

Reply:
We have the same way of displaying the content as we currently have on prod, but IMO it's not very well implemented:
Under 1280px we start to let the content being display underneath the user sidebar, which is understandable for very small screens* but not for "medium" screens (as there is plenty of space to not display content underneath the user sidebar)

* maybe we should automatically hide the user sidebar on small screens